server/src/main/java/kr/codesquad/library/global/config/properties/InterparkProperties.java

```diff
@@ -0,0 +1,27 @@
+package kr.codesquad.library.global.config.properties;
+
+import kr.codesquad.library.global.config.properties.vo.OpenApiParameter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class InterparkProperties {
+
+    @Value("${interpark.url}")
+    private String url;
+
+    @Value("${interpark.key}")
+    private String key;
+
+    @Value("${interpark.output}")
+    private String output;
+
+    @Value("${interpark.queryType}")
+    private String queryType;
+
+    @Bean
+    public OpenApiParameter OpenApiParameter() {
+        return new OpenApiParameter(url, key, output, queryType);
+    }
+}
```

Comment:
이미 Property 로 Configuration 을 했는데 OpenApiParameter 라는 Bean 을 또 만들 필요가 있을까요?

```suggestion
@Getter
@Setter
@Configuration
@ConfigurationProperties(prefix = "interpark")
public class InterparkProperties {

    private String url;
    private String key;
    private String output;
    private String queryType;
}
```

Reply:
만약 이렇게 하면 이 클래스를 어떻게 사용하나요? `new InterparkProperties` 로 해야하나요?

Reply:
Property 는 지속적으로 쓰이는 것이 아니기에 주로 생성자에서 Setup 만 해주고 쓰이지 않습니다.
만약 지속적으로 사용해야된다면 저 Property 자체를 Autowired 할것 같지만, 그렇지 않도록 설계할 것 같습니다.